app/templates/components/password/reset-password.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+  <form>
+    {{input autofocus="true" class="has-progress" name="password" type="password" placeholder='Enter new password' autocomplete="off" value=password}}
+
+    {{input class="has-progress" name="password" type="password" placeholder='Confirm password' autocomplete="off" value=passwordConfirmation}}
```

Comment:
Yeah, I don't think passwords should typically have a placeholder since it's a little confusing UX.